src/Uno.UI/UI/Xaml/Media/ImageSource.crossruntime.cs

```diff
@@ -160,7 +160,7 @@ private void OnOpened(ImageData data)
 		{
 			_imageData = data; // We should also cache the targetWidth and targetHeight
 
-			if (this.Log().IsEnabled(LogLevel.Information))
+			if (this.Log().IsEnabled(LogLevel.Debug))
```

Comment:
I know this works, but would not make more sense also to print "Debug" the data?
```
this.Log().Debug($"Image {this} opened with {data}");
```

Reply:
Oh definitely! My bad!